src/components/MyShareCard.tsx

```diff
@@ -46,7 +50,12 @@ function MyShareCard({ data }: Props): ReactElement | null {
         </div>
         <div className="infoItem">
           <span className="label bold">{`${t("totalAmount")}: `}</span>
-          <span className="value">{formattedData.value}</span>
+          <span className="value">{formattedData.amount}</span>
+          {data.amountStakedOnKeep.gt(Zero) ? (
+            <span className="value">
+              &nbsp;({formattedData.amountStakedOnKeep} on KEEP)
```

Comment:
nit: i think saying `staked` (maybe with a hyperlink to the rewards dashboard) is more clear than `on KEEP`, also could we add a translation for this

Reply:
+1 on hyperlink to the rewards dashboard